xcodeproj/internal/xcodeproj_runner.bzl

```diff
@@ -570,6 +572,8 @@ xcodeproj_runner = rule(
         "tvos_simulator_cpus": attr.string(),
         "unfocused_labels": attr.string_list(default = []),
         "unowned_extra_files": attr.string_list(),
+        "visionos_device_cpus": attr.string(mandatory = True),
+        "visionos_simulator_cpus": attr.string(mandatory = True),
```

Comment:
👋 Mind clarifying why the other `*_simulator_cpus` are not mandatory and this one is?

Reply:
Mandatory means that it has to have a value. The others don't need a value because `None` is used to determine the simulator CPU based on the host CPU. That doesn't happen for visionOS, since it only has an arm64 simulator.

Reply:
Oh right, that makes sense to me, thx I just wanted to sanity check I was understanding the changes